container/rkt/helpers.go

```diff
@@ -20,17 +20,56 @@ import (
 	"path"
 	"strings"
 
+	rktapi "github.com/coreos/rkt/api/v1alpha"
 	"github.com/golang/glog"
+	"golang.org/x/net/context"
 )
 
 type parsedName struct {
 	Pod       string
 	Container string
 }
 
-func verifyName(name string) (bool, error) {
-	_, err := parseName(name)
-	return err == nil, err
+func verifyPod(name string) (bool, bool, error) {
+	pod, err := cgroupToPod(name)
+
+	if err != nil || pod == nil {
```

Comment:
`nil, nil` sounds like an unexpected return. At the very least a doc-comment about it would be nice, but it would be nice to not have to at all. Since it's not exported, I won't bang too hard on it.